benchmarks/src/jmh/java/com/linecorp/armeria/shared/EventLoopJmhExecutor.java

```diff
@@ -53,8 +53,8 @@ public static EventLoop currentEventLoop() {
         return CURRENT_EVENT_LOOP.get();
     }
 
-    public EventLoopJmhExecutor(int numThreads, String threadPrefix) {
-        super(numThreads, new EventLoopThreadFactory(threadPrefix));
+    public EventLoopJmhExecutor(int numThreads, String threadNamePrefix) {
+        super(numThreads, ThreadFactories.builder(threadNamePrefix).eventLoop(true).build());
```

Comment:
I renamed it for consistency.
Over the project, the name `threadNamePrefix` is used.
Here was the only one where `threadPrefix` is used except newly added code.